parm/FV3LAM_wflow.xml

```diff
@@ -1188,6 +1189,46 @@ the <task> tag to be identical to the ones above for other output times.
   {%- endif %}
 {%- endif %}
 {%- endif %}
+{%- if run_task_run_prdgen %}                      
```

Comment:
Just checking...is this only meant to run for the hourly graphics (as opposed to subhourly)? _Yes_ is totally fine, just curious.

Reply:
Eventually we will likely want to run the prdgen job for subhourly output too, but initially with this PR I think it is fine to only include it for hourly output.